components/sandbox/index.js

```diff
@@ -162,8 +174,11 @@ export default class Sandbox extends Component {
 					<style dangerouslySetInnerHTML={ { __html: style } } />
 				</head>
 				<body data-resizable-iframe-connected="data-resizable-iframe-connected" className={ this.props.type }>
-					<div dangerouslySetInnerHTML={ { __html: this.props.html } } />
+					<div id="content" dangerouslySetInnerHTML={ { __html: this.props.html } } />
```

Comment:
why do we need a separate div for the html, js and styles can't we just concat everything when passing the html prop?